### ISSUE TITLE
Implement Fast Sync (UTXO Set snapshot)

### ISSUE THREAD

**Issue by ffrediani** [getumbrel/umbrel-bitcoin#21]
Can Fast Sync be implemented on Umbrel Bitcoin in order to accelerate initial blockchain synchronization and allow users to start using some applications that are fine to use a in this way or when using a pruned node ?

How can UTXO Set snapshot can be implemented or at least be optional on Umbrel Bitcoin web interface to allow the user to choose it specifically in such cases ?

Reference: https://docs.btcpayserver.org/Docker/fastsync/

**Comment by fjahr** [comment]
I would recommend to implement support for AssumeUTXO instead, which will be available for mainnet use starting with Bitcoin Core v28 which should be available some time in October: https://github.com/bitcoin/bitcoin/issues/29891 I expect that BTCPayServer replaces Fast Sync with AssumeUTXO at some point in the future too.

**Comment by yshwanth** [comment]
https://blog.lopp.net/bitcoin-node-sync-with-utxo-snapshots/ 
How do we do this in Umbrel for fast sync and is there any guide ?

**Comment by fjahr** [comment]
> https://blog.lopp.net/bitcoin-node-sync-with-utxo-snapshots/ How do we do this in Umbrel for fast sync and is there any guide ?

Why do you want to use fast sync and not assumeutxo? There is up-to-date documentation on assumeutxo: https://github.com/bitcoin/bitcoin/blob/master/doc/assumeutxo.md and the next bitcoin core release (v29) will allow a mainnet snapshot for 880,000: https://github.com/bitcoin/bitcoin/pull/31969

**Comment by ffrediani** [comment]
That sounds a quiet interesting thing to have on Umbrel UX certainly.

> https://blog.lopp.net/bitcoin-node-sync-with-utxo-snapshots/ How do we do this in Umbrel for fast sync and is there any guide ?

**Comment by yshwanth** [comment]
Yes, we need assumeutxo in umbrel for low hardware devices(Pi,..) to background sync IBD after loading/sync assumeutxo dat file.  Waiting for a week to sync IBD and do Lightning (LND/CoreLN) for new users is let down

**Comment by yshwanth** [comment]
> There is up-to-date documentation on assumeutxo: https://github.com/bitcoin/bitcoin/blob/master/doc/assumeutxo.md and the next bitcoin core release (v29) will allow a mainnet snapshot for 880,000: [bitcoin/bitcoin#31969](https://github.com/bitcoin/bitcoin/pull/31969)

Steps 
- Download the magnet link using qBittorrent 
- Copy from utxo880k.dat file in Downloads, paste inside bitcoin data folder
- Go to terminal inside bitcoin docker, execute those commands 
- Check troubleshoot for logs and successful chaintip
- Check Bitcoin app to sync from 880000 onwards...!
